sdk/contentunderstanding/azure-ai-contentunderstanding/generated_samples/content_analyzers_delete_analyzer.py

```diff
@@ -0,0 +1,90 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------
+"""
+Async sample: delete a custom analyzer using the delete API.
+
+Prerequisites:
+    pip install azure-ai-contentunderstanding python-dotenv
+    az login  # or set AZURE_CONTENT_UNDERSTANDING_KEY
+
+Environment variables:
+    AZURE_CONTENT_UNDERSTANDING_ENDPOINT   (required)
+    AZURE_CONTENT_UNDERSTANDING_KEY        (optional – falls back to DefaultAzureCredential)
+
+Run:
+    python content_analyzers_delete_analyzer.py
+"""
+
+from __future__ import annotations
+
+import asyncio
+import os
+
+from dotenv import load_dotenv
+from azure.ai.contentunderstanding.aio import ContentUnderstandingClient
+from azure.ai.contentunderstanding.models import (
+    ContentAnalyzer,
+    ContentAnalyzerConfig,
+    FieldSchema,
+    FieldDefinition,
+    FieldType,
+    GenerationMethod,
+)
+from sample_helper import get_credential
+
+load_dotenv()
+
+
+# ---------------------------------------------------------------------------
+# Sample: Delete custom analyzer using delete API
+# ---------------------------------------------------------------------------
+# This sample demonstrates:
+# 1. Authenticate with Azure AI Content Understanding
+# 2. Create a custom analyzer (for deletion demo)
+# 3. Delete the analyzer using the delete API
+# 4. Verify the analyzer is no longer available
+
+async def main() -> None:
+    endpoint = os.environ["AZURE_CONTENT_UNDERSTANDING_ENDPOINT"]
+    credential = get_credential()
+
+    async with ContentUnderstandingClient(endpoint=endpoint, credential=credential) as client, credential:
```

Comment:
If we use scenario-based samples, we wouldn't have so much similarity between create_or_replace and delete_analyzer.  In this case, I feel more samples actually hurt.